sdk/src/main/java/com/ingresse/sdk/builders/URLBuilder.kt

```diff
@@ -26,9 +29,15 @@ class URLBuilder(host: Host, environment: Environment = Environment.PROD) {
     private val hostPrefix = "https://"
 
     init {
-        val hmls = listOf(Environment.HML, Environment.HML_A, Environment.HML_B)
+        val hmls = listOf(Environment.HML,
+                Environment.HML_A,
+                Environment.HML_B,
+                Environment.HML_C)
+
         url = hostPrefix + environment.prefix + host.address
 
+        if (host == Host.LIVE || host == Host.LIVE_HML) { url = hostPrefix + host.address }
```

Comment:
Aqui voce tem que verificar se o host é live e o env é de hml pra mudar a url
```suggestion
       if (host == Host.LIVE && hmls.contains(environment)) {
            url = hostPrefix + Host.LIVE_HML.address
        }
```